### PR TITLE
Changes to better support Swift REPL

### PR DESCRIPTION
**Pull request by kdawgwilk**
Currently the main issues with using the Swift REPL is that we have to do quite a bit of setup code once we get the REPL started up.  This simplifies that into a bootstrap function and adds a dynamic library to be linked with the REPL

### CODE REVIEW

File: Sources/App/bootstrap.swift [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import Service
+import Vapor
+import Foundation
+
+public func bootstrap() throws -> Application {
```
Comment (tanner0101) [inline Sources/App/bootstrap.swift:5]: A comment here explaining what this method does and where it's called from would probably be very appreciated!

**Review by tanner0101 — APPROVED** [review]
this looks great

**Comment by tanner0101** [comment]
@kdawgwilk did you put the command you used to run the repl somewhere?

**Comment by 0xTim** [comment]
Closing as this repo has been superseded by vapor/template - feel free to reopen on that repository if this PR still makes sense. Thanks for contributing!

**Pull request closed without merging**